src/Compilers/VisualBasic/Portable/Parser/ParseConditional.vb

```diff
@@ -248,7 +251,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
         Private Function ParseRegionDirective(hashToken As PunctuationSyntax) As RegionDirectiveTriviaSyntax
             Debug.Assert(CurrentToken.Kind = SyntaxKind.IdentifierToken AndAlso DirectCast(CurrentToken, IdentifierTokenSyntax).PossibleKeywordKind = SyntaxKind.RegionKeyword,
-                         "ParseRegionDirective called with wrong token.")
+                         $"{NameOf(ParseRegionDirective)} called with wrong token.")
```

Comment:
Check out the code difference:
```
    Sub Test1(p1 As String)
        System.Console.WriteLine(NameOf(p1) & "a")
    End Sub

    Sub Test2(p2 As String)
        System.Console.WriteLine($"{NameOf(p2)} & a")
    End Sub
```

```
.method public instance void  Test1(string p1) cil managed
{
  // Code size       13 (0xd)
  .maxstack  8
  IL_0000:  nop
  IL_0001:  ldstr      "p1a"
  IL_0006:  call       void [mscorlib]System.Console::WriteLine(string)
  IL_000b:  nop
  IL_000c:  ret
} // end of method Class1::Test1

.method public instance void  Test2(string p2) cil managed
{
  // Code size       23 (0x17)
  .maxstack  8
  IL_0000:  nop
  IL_0001:  ldstr      "{0} & a"
  IL_0006:  ldstr      "p2"
  IL_000b:  call       string [mscorlib]System.String::Format(string,
                                                              object)
  IL_0010:  call       void [mscorlib]System.Console::WriteLine(string)
  IL_0015:  nop
  IL_0016:  ret
} // end of method Class1::Test2
```

Using a literal vs. String.Format call when there is nothing to format.